mlir/lib/Dialect/Rock/Transforms/LayoutEmitter.cpp

```diff
@@ -0,0 +1,64 @@
+//===- LayoutEmitter.cpp - MLIR helper that contains the layout logic -===//
```

Comment:
nit (naming) : this could be GridLayoutEmitter as it not a generic layout emitter